thisdot/lets-chat-with
# Fix/23 profile social links

# Overview

Makes social links editable and clickable.

# Screenshots for Mobile and Desktop views (if applicable)

# Details

## General

- Adds social link validation package
- Updates edit profile inputs to sanitize copied and default links
- Makes profile card social links clickable

### Manual Testing

Write the steps required for how might test this. Example:

1. Sign in
2. Enter a conference
3. Select Profile
4. Edit profile
5. Enter an FQDN social link
6. Save the form
7. Edit profile
8. Save again (double-check the URL sticks)
